.circleci/config.yml

```diff
@@ -153,21 +147,26 @@ workflows:
   build:
     jobs:
       - build:
+          context: npn-publish
           filters:
             branches:
               ignore:
                 - /^(R|r)elease.*$/
   build-release:
     jobs:
       - build:
+          context: npn-publish
           filters:
             branches:
               only:
                 - /^(R|r)elease.*$/
       - release:
-          context: github-fin-eng-context
+          context: 
+            - github-fin-eng-context
+            - npn-publish
           requires:
             - build
           filters:
             branches:
               only:
+                - /^(R|r)elease.*$/
```

Comment:
For my understanding, does this filter on github branches? So we would need to create a release.XXX branch when bumping the version

Reply:
We moved from releasing from main to releasing from a branch. So you branch from what you want to release and as long as the the branch name is something like "release-1.0.0" then it'll pick up and run the release job. The release job will check the `daml.yaml` file in each package to check if a tag exists; When it doesn't, it'll release that package.